plugins/woocommerce/src/Internal/Features/FeaturesController.php

```diff
@@ -312,6 +312,26 @@ private function get_feature_definitions() {
 						'woocommerce'
 					),
 				),
+				'blueprint'              => array(
+					'name'               => __( 'Blueprint (Beta)', 'woocommerce' ),
```

Comment:
```suggestion
					'name'               => __( 'Blueprint (beta)', 'woocommerce' ),
```

Apologies, could we make this sentence case?